repository/src/main/java/uk/gov/hmcts/ccd/definition/store/repository/model/AccessTypeRolesRoleResult.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.hmcts.ccd.definition.store.repository.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel(description = "role")
+public class AccessTypeRolesRoleResult {
+
+    private String caseTypeId;
+    private String organisationalRoleName;
+    private String groupRoleName;
+    private String caseGroupIdTemplate;
+
```

Comment:
```suggestion
private String groupAccessEnabled;
```

I think we may also need `groupAccessEnabled` (not specified in Jira though)
but HLD suggests it's used in PRM to decide whether to create a ORG or CASE role